spark-cassandra-connector/scala-2.11/src/main/scala/com/datastax/spark/connector/rdd/reader/AnyObjectFactory.scala

```diff
@@ -0,0 +1,194 @@
+package com.datastax.spark.connector.rdd.reader
```

Comment:
I'd certainly like to *not* have to maintain this class separately for scala 2.10 and 2.11. Seems there is exactly *one* single line change in this file - the line to get the constructor declaration. Please extract the method for getting constructor into external object, that can be coded differently for 2.10 and 2.11.